Dockerfile

```diff
@@ -52,12 +52,10 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 # Add PHP-settings
 ADD php-conf.d/ $PHP_INI_DIR/conf.d/
 
-# Create private-files volume, copy sites/default's defaults and make it a volume
+# Create private-files folder, copy sites/default's defaults
 RUN mkdir private && chown -R www-data:www-data /var/www
```

Comment:
Not sure I understand.
if we don't explicitly make that folder a volume in the Dockerfile, why leave the comment ?